### ISSUE TITLE
How to exclude third party dependency's APIs from sneaking into our application Swagger documentation?

### ISSUE THREAD

**Issue by PSrinivas0813** [swagger-api/swagger-ui#10214]
### Q&A (please complete the following information)
 - OS: [e.g. macOS] Windows
 - Browser: [e.g. chrome, safari] Chrome
 - Version: [e.g. 22]
 - Method of installation: [e.g. npm, dist assets]
 - Swagger-UI version: [e.g. 3.10.0] : 4.0.0
 - Swagger/OpenAPI version: [e.g. Swagger 2.0, OpenAPI 3.0] : Open API 3.0

### Content & configuration

We have a Java EE application with Rest Endpoints. We use annotations to mark the endpoints to be picked up by Swagger UI. These endpoints do show up correctly. We use a third party library - Apache's solr-solrj, this library internally uses solr-api as a dependency because of which these APIs also show up in the Swagger documentation.

How can we exclude these Solr APIs from showing up?

Maven Dependency:
```
        <dependency>
            <groupId>org.apache.solr</groupId>
            <artifactId>solr-solrj</artifactId>
            <version>9.5.0</version>
        </dependency>
```

I have tried `resourcePackages`, `resourceClasses` but no luck.

We now have removed these APIs manually by overring the default behavior of `filterAPI `method.

There has to be a better way to do this. Can you please guide.

Swagger/OpenAPI definition:
```yaml
# your YAML here
```

Swagger-UI configuration options:
```js
SwaggerUI({
  // your config options here
})
```

```
?yourQueryStringConfig
```

**Comment by robert-hebel-sb** [comment]
We are doing some housekeeping on our issue tracker. This issue has been inactive for over a year and we have not been able to tackle it yet.

Is this still important to you? If so, please leave a comment – we'd love to know if you're still affected. If we don't hear back in 7 days, we'll close this issue to tidy up the tracker.

Thank you for your understanding!

**Comment by robert-hebel-sb** [comment]
No response received after 7 days. Closing this issue to keep the tracker tidy. If you're still affected, please reopen or create a new issue with updated details.

**Comment by pruthula** [comment]
Yes this is important change we need. Apologies for the delay in getting back to you.
